src/nginx/azext_nginx/aaz/latest/nginx/deployment/_create.py

```diff
@@ -201,10 +213,45 @@ def _build_arguments_schema(cls, *args, **kwargs):
         )
 
         scaling_properties = cls._args_schema.scaling_properties
+        scaling_properties.profiles = AAZListArg(
+            options=["profiles"],
+        )
         scaling_properties.capacity = AAZIntArg(
             options=["capacity"],
         )
 
+        profiles = cls._args_schema.scaling_properties.profiles
+        profiles.Element = AAZObjectArg()
+
+        _element = cls._args_schema.scaling_properties.profiles.Element
+        _element.capacity = AAZObjectArg(
+            options=["capacity"],
+            help="The capacity parameters of the profile.",
+            required=True,
+        )
+        _element.name = AAZStrArg(
+            options=["name"],
+            required=True,
+        )
+
+        capacity = cls._args_schema.scaling_properties.profiles.Element.capacity
+        capacity.max = AAZIntArg(
+            options=["max"],
```

Comment:
I am not sure how this looks like on Azure CLI but "min" and "max" are not good names as they don't tell the user much about what the min and max are about or applied to.

Reply:
[{name:default,capacity:{min:10,max:30}}]